src/ray/gcs/gcs_server/gcs_job_manager.cc

```diff
@@ -224,8 +224,8 @@ void GcsJobManager::HandleGetAllJobInfo(rpc::GetAllJobInfoRequest request,
                 const Status &status,
                 const rpc::NumPendingTasksReply &num_pending_tasks_reply) {
               if (!status.ok()) {
-                RAY_LOG(ERROR) << "Failed to get is_running_tasks from core worker: "
-                               << status.ToString();
+                RAY_LOG(WARNING) << "Failed to get is_running_tasks from core worker: "
```

Comment:
I had one question, how to do when caught some, or it is only warn the user should care this operation, why not INFO?